tools/dir_watch.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# Quick script to Watch a directory for new files and processes them through
+# Turbinia.  Currently assumes that files are raw (unencrypted) disk images.
+
+
+if [[ -z "$1" ]] ; then
+        echo "Usage: $0 <dir to watch>"
+        exit 1
+else
+        watchdir=$1
+fi
+
+turbiniactl="$( dirname "$0" )/../turbiniactl"
```

Comment:
I guess I could have it check to see if there is a turbiniactl in the path first, but I thought it was likely that it wasn't in the path.  This will probably be more likely once we update the setup.py (I filed an issue to update this).